frontend/src/lib/utils/utils.ts

```
@@ -156,6 +156,7 @@ export function isEmptyValue(value: unknown): boolean {
 }
 
 export const isInsForgeCloudProject = () => {
+  return true;
   return window.location.hostname.endsWith('.insforge.app');
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)